packages/router/src/__tests__/set.test.tsx

```diff
@@ -61,13 +61,27 @@ test('wraps components in other components', async () => {
           <h1>
             ChildA
           </h1>
+          <div
+            aria-atomic="true"
+            aria-live="assertive"
+            id="redwood-announcer"
+            role="alert"
+            style="position: absolute; top: 0px; width: 1px; height: 1px; padding: 0px; overflow: hidden; clip: rect(0px, 0px, 0px, 0px); white-space: nowrap; border: 0px;"
+          />
```

Comment:
@Tobbe I had to update `set.test.tsx` to include the markup for the route announcer (rendered by the `PageLoader`)--double checking that this doesn't screw things up for you?

Reply:
Yeah, I kind of wish <Set> was less tied to the router, but I had to add the stuff that checks if it has any matching children or not. So changes to the router unfortunately affects the Set component and tests as well.

Your changes look 👍

Reply:
Makes sense, and I will work on decoupling the announcement in the future, don't like it either! All the announcement really needs to know is when the page is rendered, and right now the only way to really know that is in `componentDidUpdate`